### PR TITLE
added params to auto-create secret with

### PR DESCRIPTION
**Pull request by jessesarn**
Cherry pick for 1.4 from https://github.com/integr8ly/installation/pull/590

### CODE REVIEW

**Comment by odra** [comment]
can you put the orignal pr url that is being cherry picked into the this pr description please?

**Comment by jessesarn** [comment]
> can you put the orignal pr url that is being cherry picked into the this pr description please?

done

File: roles/backup/tasks/main.yml [hunk 1]
```diff
@@ -2,8 +2,20 @@
 - name: "Create {{ backup_namespace }} namespace"
   shell: "oc create namespace {{ backup_namespace }}"
   register: backup_namespace_result
+  changed_when: backup_namespace_result.rc == 0 
   failed_when: backup_namespace_result.stderr != '' and 'AlreadyExists' not in backup_namespace_result.stderr
 
+- name: "Create/update secret with s3 credentials"
+  shell: >
+    oc create secret generic {{ aws_s3_backup_secret_name }} -n {{ backup_namespace }} \
+      --from-literal=AWS_ACCESS_KEY_ID={{ backup_s3_aws_access_key }} \
```
Comment (pat-cremin) [inline roles/backup/tasks/main.yml]: Does this create the base64 encoded version of the values? If not, testing yesterday showed the backups failing when not encoded.

Comment (jessesarn) [reply roles/backup/tasks/main.yml]: Yes, they get base64 encoded.  You can see the behavior here:
{code}
$ oc create secret generic abc --from-literal=KEY=VALUE -o yaml --dry-run
apiVersion: v1
data:
  KEY: VkFMVUU=
kind: Secret
metadata:
  creationTimestamp: null
  name: abc

$ echo VkFMVUU= | base64 -d
VALUE
{code}

**Pull request merged**